src/views/OnBoardingSubProduct/OnBoardingSubProduct.tsx

```diff
@@ -176,7 +176,7 @@ function OnBoardingSubProduct() {
       Component: () =>
         SubProductStepSelectUserParty({
           parties,
-          forward: (party?: Party) => {
+          forward: (party?: SelfcareParty) => {
```

Comment:
Party is the right model to use here
```suggestion
          forward: (party?: Party) => {
```